src/Exception/EntityNotFoundException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\Common\Exception;
+
+use Throwable;
+
+class EntityNotFoundException extends ClientException
+{
+    public function __construct(
+        string $token = ExceptionTokens::ENTITY_NOT_FOUND,
+        string $message = ExceptionTokens::ENTITY_NOT_FOUND,
```

Comment:
Tokens are `ENTITY_NOT_FOUND` for example, and a message can be anything. This was originally done to allow for further details. In shop-front, we use the token as a translation key, and only display that.

Thinking about it now, especially as we want to use translations everywhere, I think we could just use `$token` and validate it (to prevent people misusing the exceptions), and in the `parent::__construct()` in `DomainException`, send `$token` as the `$message`.

In most cases I don't think there is really a good use of the current `$message`.